wallet/bindings/capacitorjs/src/definitions.ts

```diff
@@ -3,6 +3,7 @@
 
 import type { EventType, AccountManagerOptions } from './types';
 export interface IotaWalletMobileTypes {
+    initLogger(path: string): Promise<void>;
```

Comment:
initLogger also takes a level, is this set as a default somewhere?

Reply:
yes it should use this interface as parameter https://github.com/iotaledger/iota-sdk/blob/develop/wallet/bindings/nodejs/types/loggerConfig.ts#L5